pkg/kotsclient/app.go

```diff
@@ -105,5 +105,5 @@ func (c *GraphQLClient) GetApp(appID string) (*types.App, error) {
 		}
 	}
 
-	return nil, errors.New("App not found")
```

Comment:
I don't like this (string concat in general)

I'd prefer for this to be `fmt.Errorf("app %q not found", appID)` or similar instead, but it's not a blocker